### PR TITLE
qt: Correct misleading "overridden options" label

### PR DESCRIPTION
**Pull request by hebasto**
Refs: #3867, #8165.

### CODE REVIEW

**Review by Sjors — APPROVED** [review]
Much better text imo.

utACK cddc166ceac284e71522d973733a12b2803218a8

FYI #12833 would completely eliminate the need for this message.

File: src/qt/forms/optionsdialog.ui [hunk 1]
```diff
@@ -714,7 +714,7 @@
         <item>
          <widget class="QLabel" name="overriddenByCommandLineInfoLabel">
           <property name="text">
-           <string>Active command-line options that override above options:</string>
+           <string>GUI options overridden by the command line or in the configuration file:</string>
```
Comment (laanwj) [inline src/qt/forms/optionsdialog.ui]: I think the first part is confusing, these are not only *GUI* options, but anything that can be configured in the options dialog.

Comment (hebasto) [reply src/qt/forms/optionsdialog.ui]: Could it be:
"_Options set in this dialog and overridden by the command line or in the configuration file:_" ?

Comment (laanwj) [reply src/qt/forms/optionsdialog.ui]: yes, that's better

**Comment by hebasto** [comment]
@laanwj 
Thank you for your review.
Your comment has been addressed. Would you mind re-reviewing?

**Comment by laanwj** [comment]
utACK 75143612546450c064ccd8dcf1f7f6e79f397802

**Pull request merged**